packages/tempus/src/Tempus_IntegratorBasic_impl.hpp

```diff
@@ -257,6 +248,16 @@ void IntegratorBasic<Scalar>::setTimeStepControl(
     timeStepControl_ = Teuchos::null;
```

Comment:
I think this line is superfluous and should be removed. It's not part of your change set, but maybe you could remove as part of this push?